### PR TITLE
Removed duplicated zip and release code

### PR DESCRIPTION
**Pull request by jonasdmoeller**
Remove duplicated code after refactor in dh3-environments for grouping databricks release artifacts.

### CODE REVIEW

File: .github/workflows/databricks-build-prerelease.yml [hunk 1]
```diff
@@ -64,46 +63,36 @@ jobs:
...
       - name: Create Python Wheel for Databricks Jobs
         shell: bash
         working-directory: ${{ inputs.WHEEL_WORKING_DIRECTORY }}
         run: |
           pip install wheel
           python setup.py sdist bdist_wheel
+          mv dist /artifacts/
 
       - name: Include Databricks assets in release
         if: ${{ inputs.SHOULD_INCLUDE_ASSETS }}
```
Comment (dstenroejl) [inline .github/workflows/databricks-build-prerelease.yml:84]: Is this still needed?

File: .github/workflows/databricks-build-prerelease.yml [hunk 2]
```diff
@@ -64,46 +63,36 @@ jobs:
...
-        with:
-          repo_token: ${{ secrets.GITHUB_TOKEN }}
-          automatic_release_tag: ${{ env.RELEASE_VERSION }}
-          prerelease: true
-          title: ${{ env.RELEASE_VERSION }}
-          files: |
-            ${{ env.RELEASE_FOLDER_PATH }}/${{ env.RELEASE_ZIP_FILENAME }}
-
       - name: Zip files for prerelease
         if: ${{ inputs.SHOULD_INCLUDE_ASSETS }}
```
Comment (dstenroejl) [inline .github/workflows/databricks-build-prerelease.yml:90]: Is this still needed?

File: .github/workflows/databricks-build-prerelease.yml [hunk 3]
```diff
@@ -64,46 +63,36 @@ jobs:
...
         uses: thedoctor0/zip-release@0.6.2
         with:
           type: 'zip'
           filename: ${{ env.RELEASE_ZIP_FILENAME }}
-          directory: ${{ env.RELEASE_WITH_ASSETS_FOLDER_PATH }}
+          directory: ${{ env.RELEASE_FOLDER_PATH }}
 
       - name: Create prerelease
         if: ${{ inputs.SHOULD_INCLUDE_ASSETS }}
```
Comment (dstenroejl) [inline .github/workflows/databricks-build-prerelease.yml:98]: Is this still needed?

**Review by dstenroejl — COMMENTED** [review]
Added a few comments

File: .github/workflows/databricks-build-prerelease.yml (hunk 1 above)
Comment (jonasdmoeller) [reply .github/workflows/databricks-build-prerelease.yml:84]: Yes, here it is, but I removed it elsewhere

File: .github/workflows/databricks-build-prerelease.yml (hunk 3 above)
Comment (jonasdmoeller) [reply .github/workflows/databricks-build-prerelease.yml]: No, removed

File: .github/workflows/databricks-build-prerelease.yml (hunk 2 above)
Comment (jonasdmoeller) [reply .github/workflows/databricks-build-prerelease.yml]: No, removed

File: .github/workflows/databricks-build-prerelease.yml
[the remark this answers is not in the document]
Comment (jonasdmoeller) [reply .github/workflows/databricks-build-prerelease.yml:1]: See other comments

**Review by dstenroejl — APPROVED** [review]

**Pull request merged**